gql/transport/httpx.py

```diff
@@ -17,6 +17,8 @@
 import httpx
 from graphql import DocumentNode, ExecutionResult, print_ast
 
+from gql.transport.data_structures import GraphQLRequest
```

Comment:
Please remove any modification from the httpx.py file. I'll explain below.

Reply:
Got it